client/src/Components/CommonComponent/Table/Table.jsx

```diff
@@ -230,32 +263,32 @@ class TableCmponent extends Component {
               key="options"
               render={(text, record) => (
                 <span>
-                    <Icon onClick = {this.props.viewValues("singleCustomer", "viewVisibility", record.key, record)}
-                      style={{
-                        fontSize: "1.2rem",
-                        color: "rgba(0, 0, 0, 0.65)"
-                      }}
-                      type="profile"
-                    />
+                  <Icon onClick={this.props.viewValues("singleCustomer", "viewVisibility", record.key, record)}
+                    style={{
+                      fontSize: "1.2rem",
+                      color: "rgba(0, 0, 0, 0.65)"
+                    }}
+                    type="profile"
+                  />
                   <Divider type="vertical" />
-                    <Icon onClick = {this.props.viewValues("singleCustomer",
-                      "editVisibilty",
-                      record.key,
-                      record)}
+                  <Icon onClick={this.props.viewValues("singleCustomer",
```

Comment:
@naremanhilles My comment above, about not actually calling the function that you are using for onClick, applies here as well.